docs/CHANGELOG

```diff
@@ -1,3 +1,34 @@
+2023-08-13    3.2.2:
+--------------------
+  * bugfixes:
+    - #13265 [BUG] Merging tools in `gridplot` leads to wrong active state next to tool icon
+    - #13288 [BUG] construct_color_bar does not support ImageStack glyph
+    - #13307 [component: tests] Incorrect file docstrings in tests/support/plugins directory
+    - #13314 [BUG] CDS stream broken for larger rollover
+
+  * tasks:
+    - #13126 [component: examples] [BUG] Mismatch in GroupFilter documentation example between text and code
+    - #13305 [component: docs] 13276 easy fixes in documentation
+    - #13326 Backports for 3.2.2
+
+2023-07-20    3.2.1:
+--------------------
+  * bugfixes:
+    - #13139 [component: bokehjs] Error in downstream library when rendering WebGL canvas with width or height of zero
+    - #13162 [component: docs] Broken link in docs/first_steps/first_steps_9.html
+    - #13248 [component: bokehjs] [BUG] Browser freezes when deleting notebook cell containing plot with TileSource
+    - #13252 [component: bokehjs] [BUG]  plots in png generated by SaveTool on gridplot layout overlap
```

Comment:
```suggestion
    - #13252 [component: bokehjs] [BUG] plots in png generated by SaveTool on gridplot layout overlap
```

Reply:
doh, I saw the approval and merged on autopilot. That said, changelogs are auto-generated on release, so updating them manually is maybe not worth it in any case.